src/core.js

```diff
@@ -1337,7 +1337,7 @@ define([
         var _this = this;
         mug.on("messages-changed", function (event) {
             _this.setTreeValidationIcon(event.mug);
-        }, null, this.data.core);
+        }, null, null, this.data.core);
```

Comment:
interesting, is this a custom event api or are you using some code from somewhere? (looks fine, just wondering)